packages/cardano-services/src/types.ts

```diff
@@ -1 +1,2 @@
 export type ModuleState = null | 'initializing' | 'initialized';
+export type Awaited<T> = T extends PromiseLike<infer U> ? U : T;
```

Comment:
This is generic, but probably not worth moving until we have implemented the `util` package.